src/components/Office/PPM/EditNetWeights/EditPPMNetWeight.jsx

```diff
@@ -86,80 +90,106 @@ const WeightCalculationHint = ({ type, firstValue, secondValue, thirdValue }) =>
 };
 
 const validationSchema = Yup.object({
-  ppmNetWeight: Yup.number().min(0, 'Net weight must be 0 lbs or greater').required('Required'),
-  ppmNetWeightRemarks: Yup.string().required('Required'),
+  adjustedNetWeight: Yup.number().min(0, 'Net weight must be 0 lbs or greater').required('Required'),
+  netWeightRemarks: Yup.string().required('Required'),
 });
 
-const EditPPMNetWeightForm = ({ onSave, onCancel, initialValues }) => (
-  <Formik initialValues={initialValues} validationSchema={validationSchema}>
-    {({ handleChange, values, isValid, errors, touched, setTouched }) => (
-      <div>
-        <Fieldset className={styles.fieldset}>
-          <MaskedTextField
-            data-testid="weightInput"
-            defaultValue="0"
-            id="ppmNetWeight"
-            name="ppmNetWeight"
-            mask={Number}
-            lazy={false}
-            scale={0}
-            signed={false} // no negative numbers
-            thousandsSeparator=","
-            suffix="lbs"
-            inputClassName={styles.weightInput}
-            errorClassName={styles.errors}
-            labelClassName={styles.weightLabel}
-          />
-          <Label htmlFor="remarks">Remarks</Label>
-          <ErrorMessage
-            className={styles.errors}
-            display={!!touched.ppmNetWeightRemarks && !!errors.ppmNetWeightRemarks}
-          >
-            {errors.ppmNetWeightRemarks}
-          </ErrorMessage>
-          <ErrorIndicator hasErrors={!!touched.ppmNetWeightRemarks && !!errors.ppmNetWeightRemarks}>
-            <Textarea
-              id="ppmNetWeightRemarks"
-              data-testid="formRemarks"
-              maxLength={500}
-              placeholder=""
-              onChange={handleChange}
-              onBlur={() => {
-                setTouched({ ppmNetWeightRemarks: true }, false);
-              }}
-              value={values.ppmNetWeightRemarks}
+const EditPPMNetWeightForm = ({ onCancel, initialValues, weightTicket }) => {
+  const queryClient = useQueryClient();
+
+  const { mutate: patchWeightTicketMutation } = useMutation({
+    mutationFn: patchWeightTicket,
+    onSuccess: () => {
+      queryClient.invalidateQueries({
+        queryKey: [DOCUMENTS],
+      });
+      onCancel();
```

Comment:
```suggestion
      onCancel();
```
This should happed where on the callback where `patchWeightTicketMutation` is called.